src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -41,6 +41,9 @@ use crate::platform_impl::x11::{
 /// The X11 documentation states: "Keycodes lie in the inclusive range `[8, 255]`".
 const KEYCODE_OFFSET: u8 = 8;
 
+/// Minimum delta for a device event to be considered.
+const NOTGULL_THRESHOLD: f32 = 4.0e-8;
```

Comment:
I should insert my username in more source code! 😁

(I've tried hard with the `mtm: MainThreadMarker`, but while `mtm` is my initials, nothing beats just spelling out the whole thing)

Reply:
Should be renamed to something useful. Also it should be explained why the given constant was picked in the first place, otherwise it _may_ not work for some cases.

There's also a case where you move the mouse really slowly, thus filtering may not actually move the mouse where it should or accumulate in error.

Reply:
I've elected to use the float epsilon instead. Since float comparison breaks down past the epsilon, for all intentions and purposes any number less than the epsilon is zero.